### PR TITLE
faq: add FAQ questions

### PR DESCRIPTION
**Pull request by CaitinChen**
@lilin90 @lhyPingcap PTAL
via https://github.com/pingcap/docs-cn/pull/751

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -194,6 +242,44 @@ Check the time difference between the machine time of the monitor and the time w
...
+
+2. If a slow query occurs, you can locate the `tidb-server` instance where the slow query is and slow query time point using Grafana and find the SQL statement information recorded in the log on the corresponding node.
+
+#### How to add the `label` configuration if `label` of TiKV was not configured when I deployed the TiDB cluster for the first time?
+
+The configuration of TiDB `label` is related to the cluster deployment architecture. It is important and is the basis for the global management and schedule of PD. If you did not configure `label` when deploying the cluster previously, you should adjust the deployment structure by manually adding the `location-labels` information using the PD management tool `pd-ctl`, for example, `config set location-labels "zone, rack, host"` you should configure based on the practical `label` level name. 
+
+For the usage of `pd-ctl`, see [PD Control Instruction](tools/pd-control.md).
+
+#### Why does the `dd` command for the disk test use the `oflag=direct` option?di
```
Comment (lilin90) [inline FAQ.md]: ?di -> ?

File: FAQ.md [hunk 2]
```diff
@@ -194,6 +242,44 @@ Check the time difference between the machine time of the monitor and the time w
...
+
+The Direct mode wraps the Write request into the I/O command and sends this command to the disk, thus to bypass the file system cache to test the real I/O Read/Write performance of the disk.
```
Comment (lilin90) [inline FAQ.md]: - delete "thus" (seems seldom used in tech docs)
- to test -> and directly test

File: FAQ.md [hunk 3]
```diff
@@ -524,6 +614,10 @@ TiKV supports calling the interface separately. Theoretically, you can take an i
 - Reduce the data transmission between TiDB and TiKV
 - Make full use of the distributed computing resources of TiKV to execute computing pushdown
 
+#### The error message `IO error: No space left on device While appending to file` is displayed.
+
+This is because the disk space is not enough. You should add nodes or enlarge the disk space.
```
Comment (lilin90) [inline FAQ.md]: should -> need to ("should" is too weak)

File: FAQ.md [hunk 4]
```diff
@@ -758,6 +860,18 @@ Yes. TiDB uses the cost-based optimizer. The cost model and statistics are const
 
 View the `Healthy` field using `show stats_healthy` and generally you need to execute `analyze` on a table when the field value is smaller than 60.
 
+#### What is the ID rule when a query plan is presented as a tree? What is the execution order for this tree?
+
+No rule exists for these IDs but the IDs are unique. When IDs are generated, a counter works and ID will plus one when one plan is generated. The execution order has nothing to do with the ID. The whole query plan is a tree and the execution process starts from the root node and the data is returned to the upper level continuously. For details about the query plan, see [Understanding the TiDB Query Execution Plan](sql/understanding-the-query-execution-plan.md).
```
Comment (lilin90) [inline FAQ.md]: ID will plus -> adds

File: FAQ.md [hunk 5]
```diff
@@ -758,6 +860,18 @@ Yes. TiDB uses the cost-based optimizer. The cost model and statistics are const
...
+
+#### In the TiDB query plan, `cop` tasks are in the same root. Are they executed concurrently?
+
+Currently the computing tasks of TiDB to two different tasks: `cop task` and `root task`.
```
Comment (lilin90) [inline FAQ.md]: to two different tasks -> **belong** to two different **types of** tasks

File: FAQ.md [hunk 6]
```diff
@@ -758,6 +860,18 @@ Yes. TiDB uses the cost-based optimizer. The cost model and statistics are const
...
 
+#### What is the ID rule when a query plan is presented as a tree? What is the execution order for this tree?
+
+No rule exists for these IDs but the IDs are unique. When IDs are generated, a counter works and ID will plus one when one plan is generated. The execution order has nothing to do with the ID. The whole query plan is a tree and the execution process starts from the root node and the data is returned to the upper level continuously. For details about the query plan, see [Understanding the TiDB Query Execution Plan](sql/understanding-the-query-execution-plan.md).
+
+#### In the TiDB query plan, `cop` tasks are in the same root. Are they executed concurrently?
+
+Currently the computing tasks of TiDB to two different tasks: `cop task` and `root task`. 
+
+`cop task` is the computing task which is pushed down to the KV end for execution distributedly; `root task` is the computing task executed for single point of execution on TiDB end.
```
Comment (lilin90) [inline FAQ.md]: -> `cop task` is the computing task which is pushed down to the KV end for distributed execution; `root task` is the computing task for single point execution on the TiDB end.

File: FAQ.md [hunk 7]
```diff
@@ -758,6 +860,18 @@ Yes. TiDB uses the cost-based optimizer. The cost model and statistics are const
...
+
+#### In the TiDB query plan, `cop` tasks are in the same root. Are they executed concurrently?
+
+Currently the computing tasks of TiDB to two different tasks: `cop task` and `root task`. 
+
+`cop task` is the computing task which is pushed down to the KV end for execution distributedly; `root task` is the computing task executed for single point of execution on TiDB end. 
+
+Generally the input data of `root task` comes from `cop task`; when `root task` processes data, `cop task` of TiKV can processes data at the same time and waits for the pull of `root task` of TiDB. Therefore, `cop` tasks can be considered as executed concurrently; but their data is of upstream and downstream relation. During the execution process, they are executed concurrently during some time. For example, the first `cop task` is processing the data in [100, 200] and the second `cop task` is processing the data in [1, 100]. For details, see [Understanding the TiDB Query Plan](sql/understanding-the-query-execution-plan.md).
```
Comment (lilin90) [inline FAQ.md]: their data is of upstream and downstream relation -> their data has an upstream and downstream relationship

File: FAQ.md [hunk 8]
```diff
@@ -790,6 +908,20 @@ The monitoring system of TiDB consists of Prometheus and Grafana. From the dashb
 
 Yes. Find the startup script on the machine where Prometheus is started, edit the startup parameter and restart Prometheus.
 
+#### Region Health monitor
+
+In TiDB 2.0, Region health is monitored in the PD metric monitor page, in which the `Region Health` monitor item shows the statistics of all the Region replica status. `miss` means shortage of replicas and `extra` means the extra replica exists. In addition, `Region Health` also shows the isolation level by `label`. `level-1` means the Region replicas are isolated physically in the first `label` level. All the Regions are in `level-0` when `location label` is not configured.
```
Comment (lilin90) [inline FAQ.md]: - monitor page -> monitoring page
- monitor item -> monitoring item

File: FAQ.md [hunk 9]
```diff
@@ -790,6 +908,20 @@ The monitoring system of TiDB consists of Prometheus and Grafana. From the dashb
...
+
+#### What is the meaning of `selectsimplefull` in Statement Count monitor?
+
+It means full table scan but the table may be a small system table.
```
Comment (lilin90) [inline FAQ.md]: may - might

File: FAQ.md [hunk 10]
```diff
@@ -790,6 +908,20 @@ The monitoring system of TiDB consists of Prometheus and Grafana. From the dashb
...
+
+#### What is the difference between `QPS` and `Statement OPS` in the monitor?
+
+The `QPS` statisctics is about all the SQL statements, including `use database`, `load data`, `begin`, `commit`, `set`, `show`, `insert` and `select`.
+
+The `Statement OPS` statistics is only about service related SQL statements, including `select`, `update` and `insert`, therefore the `Statement OPS` statistics matches the service.
```
Comment (lilin90) [inline FAQ.md]: - service related -> applications related
- matches the service -> matches the applications better

File: FAQ.md [hunk 11]
```diff
@@ -841,3 +973,7 @@ This error usually occurs when the version of TiDB does not match with the versi
 When you execute the `LOAD DATA LOCAL` statement but the MySQL client does not allow executing this statement (the value of the `local_infile` option is 0), this error occurs.
 
 The solution is to use the `--local-infile=1` option when you start the MySQL client. For example, use command like `mysql --local-infile=1 -u root -h 127.0.0.1 -P 4000`. The default value of `local-infile` is different in different versions of MySQL client, therefore you need to configure it in some MySQL clients and do not need to configure it in some others.
+
+#### ERROR 9001 (HY000): PD server timeoutstart timestamp may fall behind safepoint
+
+This error occurs when TiDB accesses PD. A worker in the TiDB background continuously queries the safepoint in PD and this error occurs if it fails to query within 100s. Generally it is because PD failure or the network problem between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](sql/error.md).
```
Comment (lilin90) [inline FAQ.md]: - when TiDB fails to access PD (not always occur; we should make it more logical)
- A worker in the TiDB background continuously queries the safepoint in PD -> A worker from PD continuously queries the safepoint in the TiDB background
- PD failure or the network problem -> the PD failure or network failure

**Comment by CaitinChen** [comment]
@lhyPingcap PTAL

File: FAQ.md [hunk 12]
```diff
@@ -242,29 +242,29 @@ Check the time difference between the machine time of the monitor and the time w
...
+You can use Docker Compose to build a TiDB cluster locally, including the cluster monitoring components. You can also customize the version and number of instances for each component. The configuration file can also be customized. You can only use this deployment method for testing and development environment. For details, see [Building the Cluster Using Docker Compose](op-guide/docker-compose.md).
 
 #### How to separately record the slow query log in TiDB? How to locate the slow query SQL statement?
 
-1. The slow query definition for TiDB is in the `conf/tidb.yml` configuration file of `tidb-ansible`. `slow-threshold: 300` is the threshold value of the slow query (unit: millisecond).
+1. The slow query definition for TiDB is in the `conf/tidb.yml` configuration file of `tidb-ansible`. The `slow-threshold: 300` parameter is used to configure the threshold value of the slow query. (unit: millisecond).
```
Comment (lilin90) [inline FAQ.md]: Delete the extra full stop after "query".

File: FAQ.md [hunk 13]
```diff
@@ -976,4 +976,4 @@ The solution is to use the `--local-infile=1` option when you start the MySQL cl
 
 #### ERROR 9001 (HY000): PD server timeoutstart timestamp may fall behind safepoint
 
-This error occurs when TiDB accesses PD. A worker in the TiDB background continuously queries the safepoint in PD and this error occurs if it fails to query within 100s. Generally it is because PD failure or the network problem between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](sql/error.md).
+This error occurs when TiDB fails to access PD. A worker in the TiDB background continuously queries the safepoint in PD and this error occurs if it fails to query within 100s. Generally it is because the PD failure or network failure between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](sql/error.md).
```
Comment (lilin90) [inline FAQ.md]: A worker in the TiDB background continuously queries the safepoint in PD -> **A worker continuously queries the safepoint from PD in the TiDB background**

**Comment by CaitinChen** [comment]
@lhyPingcap PTAL

**Comment by lhyPingcap** [comment]
LGTM

**Pull request merged**